modules/ROOT/pages/manage-host-certificates.adoc

```diff
@@ -0,0 +1,92 @@
+= Managing Host Certificates
+:page-aliases: configure-global-as2-settings.adoc
+
+Anypoint Partner Manager supports the following types of host certificates:
+
+[%header%autowidth.spread]
+|===
+|Certificate Type |Description | Used by
+| Identity key a| Private key used for authentication a| The following types of endpoints, when the authentication type is `Identity Key` or `Basic and Identity Key`:
+
+* SFTP Receive from Partners endpoints owned by the host
+* SFTP Source at Host endpoints
+* SFTP Send to Partners endpoints owned by the host
+* SFTP Target at Host endpoints
+| Public certificate | Certificate generated by a certificate authority (CA) a| 
+* HTTPS Send to Partners endpoints owned by the host, when the TLS context is set to `Use a self-signed certificate`
+* HTTPS Target at Host endpoints, when the TLS context is set to `Use a self-signed certificate`
+|===
+
+NOTE: For AS2 endpoints, you must also configure at least one host keystore, as described in xref:manage-as2-host-keystores.adoc[]. 
```

Comment:
you must also configure > configure

Reply:
Leaving as is because it is in addition to the host certificates.